models/qualtrics__contact.sql

```diff
@@ -54,8 +54,8 @@ agg_distribution_responses as (
     select 
         contact_id,
         source_relation,
-        -- should i include other distribution channels? qr code, social media, personal links, etc?
-        -- should i add avg time-to-open?
+        -- should i include other distribution channels? qr code, social media, personal links, etc? 
+        -- should i add avg time-to-open? 
```

Comment:
I think that would be an interesting metric to gauge how active a contact is. What is the LOE to add that in? If it is not too much I say include it, if it is a large lift then let's exclude for now as it is not crucial.

Reply:
ah yeah i'll just go ahead and remove